app/i18n/locales/en.json

```diff
@@ -744,5 +744,6 @@
   "accept": "Accept",
   "Incoming_call_from": "Incoming call from",
   "Call_started": "Call started",
-  "Message_has_been_shared":"Message has been shared"
+  "Message_has_been_shared":"Message has been shared",
+  "No_channels_in_team": "No Channels on this Team"
```

Comment:
on this PR https://github.com/RocketChat/Rocket.Chat.ReactNative/pull/5056 was removed unnecessary uppercase, I think we should keep this behavior and change the `Team` to `team`

Reply:
Also, IMHO the "in" is more correct than "on" 😅

Reply:
in and on have different meanings depending on the context, in this context the meaning would be more or less like this:
in: Não existem canais nesse time.
on: Esse time não tem canais.

Literally translating it sounds weird.